build-scripts/config_common.cmake

```diff
@@ -464,7 +464,13 @@ else ()
 endif ()
 
 if (APPLE)
+  EXEC_PROGRAM(sw_vers ARGS -productVersion OUTPUT_VARIABLE OS_X_VERSION)
```

Comment:
i guess we should check cmake target instead.

Reply:
you mean like here? https://github.com/neovim/neovim/blob/3fef24aa01ab2223d5455e7b22d218ea91495c86/cmake.deps/CMakeLists.txt#L61-L66